File: python/properties-methods-and-instances/exercise02/classStudent.py
class Student:
  def __init__(self, nombre, edad, concentracion, promedio):
    self.nombre = nombre
    self.edad = edad
    self.concentracion = concentracion
    self.promedio = promedio

  def mostrarDetalles(self):
    print(f"1. Nombre: {self.nombre}")
    print(f"2. Edad: {self.edad}")
    print(f"3. Concentración: {self.concentracion}")
    print(f"4. Promedio: {self.promedio}")

File: javascript/properties-methods-and-instances/exercise02/classStudent.js
class Student {
  constructor(nombre, edad, concentracion, promedio) {
    this.nombre = nombre;
    this.edad = edad;
    this.concentracion = concentracion;
    this.promedio = promedio;
  }

  // Metodo
  mostrarDetalles() {
    console.log(`Nombre: ${this.nombre}`);
    console.log(`Edad: ${this.edad}`);
    console.log(`Concentracion: ${this.concentracion}`);
    console.log(`Promedio: ${this.promedio}`);
  }
}

module.exports = Student;